frontend/src/app/shared/components/file-links/file-link-list-item/file-link-list-item.html

```diff
@@ -1,6 +1,9 @@
-<div class="spot-list--item-floating-wrapper">
+<div
+  class="spot-list--item-floating-wrapper op-files-tab--file-list-item-floating-wrapper"
+  [ngClass]="{ 'no-hover': disabled }"
```

Comment:
That is not a BEM style class. Please use something like `op-files-tab--file-list-item-floating-wrapper_disabled` instead.

Reply:
changed to BEM